bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/controllers/requests/SignUpRequest.java

```diff
@@ -14,6 +16,7 @@ public class SignUpRequest {
 	private String firstName;
 	private String lastName;
 	private String password;
+	private Instant createdAt;
```

Comment:
Same thing here. We shouldn't be accepting a `createdAt` in the HTTP request, that's defined in our Java code with `Instant.now()` or the SQLite database